### PR TITLE
Add Rnafusion alignment file to Scout uploads

### PR DESCRIPTION
**Pull request by ivadym**
### Added
- Rnafusion alignment cram to Scout uploads

### How to prepare for test

- [ ] Ssh to relevant server (depending on type of change)
- [ ] Use stage: `us`
- [ ] Paxa the environment: `paxa`
- [ ] Install on stage (example for Hasta):
    ```shell
    bash /home/proj/production/servers/resources/hasta.scilifelab.se/update-tool-stage.sh -e S_cg -t cg -b [THIS-BRANCH-NAME] -a
    ```

### How to test

- [x] `cg workflow rnafusion store <case_id>`
- [x] `cg upload -c <case_id>`

### Expected test outcome

- [x] CRAM files stored in HK with sample ID tag 
- [x] `scout_upload.yaml` file contains the RNA alignment path
- [x] Alignment file visualised in IGV in Scout

## Review

- [ ] Tests executed by
- [ ] "Merge and deploy" approved by
Thanks for filling in who performed the code review and the test!

### This [version](https://semver.org/) is a

- [ ] **MAJOR** - when you make incompatible API changes
- [ ] **MINOR** - when you add functionality in a backwards compatible manner
- [ ] **PATCH** - when you make backwards compatible bug fixes or documentation/instructions

## Implementation Plan

- [ ] Document in ...
- [ ] Deploy this branch on ...
- [ ] Inform to ...

### CODE REVIEW

File: cg/constants/scout_upload.py [hunk 1]
```diff
@@ -86,4 +86,6 @@ class ScoutCustomCaseReportTags(StrEnum):
 )
 
 
-RNAFUSION_SAMPLE_TAGS = {}
+RNAFUSION_SAMPLE_TAGS = dict(
+    alignment_file={"cram"},
```
Comment (ChrOertlin) [inline cg/constants/scout_upload.py:90]:
```suggestion
    alignment_file={AlignmentFileTag.CRAM},
```

File: cg/meta/upload/scout/rnafusion_config_builder.py [hunk 2]
```diff
@@ -56,12 +60,20 @@ def _include_file(self, scout_key) -> None:
...
 
+    def include_sample_alignment_file(self, config_sample: ScoutIndividual) -> None:
+        """Include the RNA sample alignment file."""
+        config_sample.rna_alignment_path = self.get_sample_file(
+            hk_tags=self.sample_tags.alignment_file, sample_id=config_sample.sample_id
+        )
+
+    def include_sample_files(self, config_sample: ScoutIndividual) -> None:
+        """Include all files that are used on RNA sample level in Scout"""
+        return None
```
Comment (ChrOertlin) [inline cg/meta/upload/scout/rnafusion_config_builder.py:71]: I don't understand the purpose of this function, why does it only return None?

**Review by ChrOertlin — APPROVED** [review]
Looks good, confused about one funktion

File: cg/meta/upload/scout/rnafusion_config_builder.py (hunk 2 above)
Comment (ivadym) [reply cg/meta/upload/scout/rnafusion_config_builder.py]: I changed the default behaviour of the parent class to return None, then we don't have to include this one 👍 

Thanks for the review @ChrOertlin ⭐

**Pull request merged**